client/modules/IDE/components/Editor.jsx

```diff
@@ -346,25 +346,23 @@ class Editor extends React.Component {
         }
       );
       this._cm.doc.setValue(formatted);
-      return cursorOffset;
+      if (cursorOffset) {
+        this._cm.doc.setCursor(this._cm.doc.posFromIndex(cursorOffset));
+      }
     } catch (error) {
-      return null;
+      console.error(error);
```

Comment:
coming to the error part, I thought of replicating the original functionality. As the syntax error is only shown when we save but not when we tidy the code.

Reply:
I think it's okay to show syntax errors when tidying the code! I think it makes more sense, anyway.